pennylane/operation.py

```diff
@@ -736,14 +736,13 @@ def get_eigvals(self):
         Returns:
             tensor_like: eigenvalues
         """
-
         try:
             return self.compute_eigvals(*self.parameters, **self.hyperparameters)
         except EigvalsUndefinedError:
             # By default, compute the eigenvalues from the matrix representation.
             # This will raise a NotImplementedError if the matrix is undefined.
             try:
-                return np.linalg.eigvals(self.get_matrix())
+                return qml.math.linalg.eigvals(self.get_matrix())
```

Comment:
a bug fix I noticed!

Reply:
Is `eigvals` implemented in all autodiff frameworks?

Reply:
yep 🙂